scss/_patterns_lists.scss

```diff
@@ -186,18 +187,24 @@ $spv-list-item--inner: null;
           width: $bullet-width;
         }
       }
+    }
 
-      & + .p-stepped-list__content {
-        @media (max-width: $breakpoint-heading-threshold) {
-          margin-left: $bullet-width-mobile + $sph-inner;
-        }
+    h#{$i}.p-stepped-list__title + .p-stepped-list__content {
```

Comment:
Why does this require adjacent sibling selector? This could complicate the specificity.

Reply:
It looks like the use of `$i` is outside of the for loop

Reply:
> It looks like the use of `$i` is outside of the for loop

No. Loop ends in line 201 if I look at nesting correctly.

Reply:
@anthonydillon github has omitted some lines, which makes it look outside the loop. But the brace on line 190 closes an opening brace on line 177:
![image](https://user-images.githubusercontent.com/2741678/82305543-aaafce00-99b5-11ea-9c37-7573650427c5.png)

As to why this is needed - it's a very manual way of creating a hanging indent that works for any heading size (that's what is currently implemented, I'm not adding it with this pr). Feels like there should be a simpler way to do this so the pseudo element pushes anything underneath in line with the heading it sits next to. But that's a separate exploration.